doc/man3/DEFINE_STACK_OF.pod

```diff
@@ -297,6 +296,9 @@ and B<sk_I<TYPE>_find_all>() calls are read-only and do not sort the
 stack.  To avoid any performance implications this change introduces,
 B<sk_I<TYPE>_sort>() should be called before these find operations.
 
+Before OpenSSL 3.3.0 B<sk_I<TYPE>_push>() returned -1 if I<sk> was NULL. It
+was changed to return 0 in this condition as for other errors.
```

Comment:
I think it would be okay (but probably not required) to say why the change was made (consistency with insert and render non-buggy existing checks for equality).